gensim/models/keyedvectors.py

```diff
@@ -1063,6 +1067,134 @@ def get_keras_embedding(self, train_embeddings=False):
         )
         return layer
 
+    def simple_average(self, sent):
+        """
+        Mixture of word-vectors to make sentence-vector.
+
+        Attains this by simple averaging of all word-vectors.
+
+        Examples
+        --------
+        >>> sent = iter([['sample', 'sentence', 'cat', 'dog'], ['time', 'computers', 'expensive']])
+        >>> trained_model.simple_average(sent)
+
+
+        Parameters
+        ------------
+        sent: iterator of list of tokens of sentences
+            Corpus of sentences.
+
+        Returns
+        --------
+        sents_emd: class: `gensim.models.keyedvectors.Word2VecKeyedVectors`
+            Sentence embeddings of each sentence in the same order as given.
+
+        """
+        sents_emd = []
+        for s in sent:
+            sent_emd = []
+            for w in s:
+                if w in self.vocab:
+                    sent_emd.append(self.vocab[w])
+            sent_emd_ar = np.array(sent_emd)
+            sum_ = sent_emd_ar.sum(axis=0)
+            result = sum_ / np.sqrt((sum_ ** 2).sum())
+            sents_emd.append(result)
+        return sents_emd
+
+    def smooth_inverse_frequency(self, sent, a=0.001):
```

Comment:
maybe ask the user about dictionary too?

Reply:
did you mean pre-trained model?